eng/precommit_local_build.py

```diff
@@ -0,0 +1,99 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Python version 3.4 or higher is required to run this script.
+
+# Use case: Runs compilation, testing, and linting for the passed artifacts and POM artifacts.
+#
+# The artifacts must exist in the module list of pom.xml in the root of azure-sdk-for-java. Artifacts may be passed in two ways:
+#
+# 1. Comma separated list of relative POM paths.
+# 2. Comma separated list of groupId:artifactId identifiers.
+#
+# The script must be run at the root of azure-sdk-for-java.
```

Comment:
Might be worth including the exact command to use to run this script and the options that can be passed to this script.